src/gui/widgets/Knob.cpp

```diff
@@ -618,12 +645,55 @@ void Knob::mousePressEvent( QMouseEvent * _me )
 
 void Knob::mouseMoveEvent( QMouseEvent * _me )
 {
-	if( m_buttonPressed && _me->pos() != m_origMousePos )
+	if( m_buttonPressed && _me->pos() != m_lastMousePos )
 	{
-		m_mouseOffset = _me->pos() - m_origMousePos;
-		setPosition( m_mouseOffset );
+		// knob position is changed depending on last mouse position
+		setPosition( _me->pos() - m_lastMousePos );
 		emit sliderMoved( model()->value() );
-		QCursor::setPos( mapToGlobal( m_origMousePos ) );
+
+		// the rest of the code updates the cursor and/or m_origMousePos
+
+		// get screen min/max values where a flip is not required yet
+		const QRect& rec =
+#if QT_VERSION >= 0x050A00
+			QApplication::screenAt(QCursor::pos())->geometry();
```

Comment:
It seems like `QApplication::screenAt(QCursor::pos())` returns a null pointer on macOS if the cursor hits the bottom of the screen.